rust_dev_preview/globalaccelerator/src/bin/globalaccelerator-helloworld.rs

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+use aws_config::meta::region::RegionProviderChain;
+use aws_sdk_globalaccelerator::{Client, Error, Region, PKG_VERSION};
+use structopt::StructOpt;
+use tokio_stream::StreamExt;
+
+#[derive(Debug, StructOpt)]
+struct Opt {
+    /// Whether to display additional information.
+    #[structopt(short, long)]
+    verbose: bool,
+}
+
+// List accelerator names and ARNs.
+// snippet-start:[globalaccelerator.rust.globalaccelerator-helloworld]
+async fn show_accelerators(client: &Client) -> Result<(), Error> {
+    let mut paginator = client.list_accelerators().into_paginator().send();
+
```

Comment:
I would like to see a quick intro statement here along the lines of "Welcome to the AWS Rust SDK Global Accelerator example!" that is printed out. If the user has no accelerators to list, the output is blank.

Reply:
Will do, thanks!